styles/readwise/catppuccin.user.css

```diff
@@ -37,9 +37,9 @@
-      background-image: url("data:image/svg+xml,@{svg}");
+      // encode function throws "Invalid 응 without number" exception, please use encodeURIComponent function in browser if this needs to be edit
```

Comment:
This is because you need to replace a color within the SVG. You left it with `fill="white"`, which A) didn't theme it / change anything and B) causes that error.

Reply:
In the future feel free to join the Catppuccin Discord, where we have a userstyles-specific thread in which we can help with issues like this. I've also updated the documentation here to clarify.